packages/server/graphql/private/mutations/connectSocket.ts

```diff
@@ -38,11 +38,15 @@ const connectSocket: MutationResolvers['connectSocket'] = async (
   const userId = getUserId(authToken)
 
   // RESOLUTION
-  const user = await dataLoader.get('users').load(userId)
+  const [user, teamMembers] = await Promise.all([
+    dataLoader.get('users').load(userId),
+    dataLoader.get('teamMembersByUserId').load(userId)
```

Comment:
Here we now have an additional roundtrip to the db on connect. This is bad especially on releases. Although the dataloader should mitigate this somewhat it's still more than double the database traffic.